hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/IndexQueueServiceTest.kt

```diff
@@ -103,6 +103,56 @@ internal class IndexQueueServiceTest(@Autowired private val objectMapper: Object
     }
   }
 
+  @Nested
+  inner class SendRefreshIndexMessage {
+    @BeforeEach
+    internal fun setUp() {
+      whenever(indexSqsClient.sendMessage(any<SendMessageRequest>())).thenReturn(CompletableFuture.completedFuture(SendMessageResponse.builder().messageId("abc").build()))
+    }
+
+    @Test
+    fun `will send populate message with index name`() {
+      indexQueueService.sendRefreshIndexMessage(GREEN)
+      verify(indexSqsClient).sendMessage(
+        check<SendMessageRequest> {
+          assertThatJson(it.messageBody()).isEqualTo(
+            """{
+          "type": "COMPARE_INDEX",
+          "index": "GREEN"
+          }
+            """.trimIndent(),
+          )
+        },
+      )
+    }
+
+    @Test
+    fun `will ignore index name`() {
```

Comment:
Are these two tests (one above) exactly the same?

Reply:
yes I guess so, will fix in next PR 😂